rs/src/management/http_client.rs

```diff
@@ -472,12 +451,14 @@ pub struct TunnelClientBuilder {
 /// Creates a new tunnel client builder. You can set options, then use `into()`
 /// to get the client instance (or cast automatically).
 pub fn new_tunnel_management(user_agent: &str) -> TunnelClientBuilder {
+    let pkg_version = PKG_VERSION.unwrap_or("unknown");
+    let full_user_agent = format!("{}{}{}", user_agent, " Visual-Studio-Tunnel-Service-Rust-SDK/", pkg_version);
```

Comment:
Probably want to put a space or some kind of separator after the consumer-provided user agent.